core/server/worker/src/main/java/alluxio/worker/block/BlockWorkerDB.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.worker.block;
+
+import static alluxio.Constants.CLUSTERID_FILE;
+
+import alluxio.conf.AlluxioConfiguration;
+import alluxio.conf.PropertyKey;
+import alluxio.conf.ServerConfiguration;
+import alluxio.util.io.PathUtils;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.annotation.concurrent.NotThreadSafe;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.UUID;
+
+/**
+ * Persistent info for worker.
+ */
+@NotThreadSafe
+public interface BlockWorkerDB {
```

Comment:
`WorkerMetaStore`?

Reply:
There are already classes with similar names, Will it cause confusion? such as: 
```java
alluxio.master.block.meta.WorkerMetaLockSection
alluxio.master.block.meta.MasterWorkerInfo#lockWorkerMeta
```

Reply:
I think this is fine. `WorkerMetaStore` suggests it's a storage of worker metadata. `WorkerMeta` suggests it's worker metadata.

@beinan We are storing the clusterID on the worker locally so the worker remembers clusterID on restart. Do you think `WorkerMetaStore` is a fair name? Do you anticipate the worker to store more similar metadata locally? Do you have preference on where the worker should store those metadata (like we are putting them under `${ALLUXIO_HOME}/metadata` now i believe, instead of in the tiered storage)?

Reply:
is this class keeping the meta data of the block level meta or worker level meta?   
if it's worker level, I agree with you, I might use alluxio.worker.meta.WorkerMetaStore or alluxio.worker.WorkerMetaStore
If it's block level, it might be more complicated, because @yyongycy and I are both working some block level meta data, I think we could consolidate these metadata somehow

Reply:
Ok, I see, it's a worker level metadata, so I think WorkerMetaStore is a decent name.  But I think it's a little bit over kill to make a metadata store just for a cluster_id, we could just load the cluster_id from a local file on its name e.g.   /worker/cluster_id/000765  (the file is empty, we could get everything we need from the filename).

Reply:
@beinan I don't think there is a big difference between using the file name and the file content, because the IO of this file only happens when the worker starts and registers, which is an infrequent operation.
Let the user know the function of the file through the file name and some comments

Reply:
@xichen01 I'm not talking about the performance, I'm just thinking if it's overkilled by adding a couple of classes just for reading a cluster id from the disk.

Reply:
Yea if we have a complicated design here, that'd be an overkill because it's a design without real use cases.

For the moment I think it's fine. The `BlockWorkerDB`(to be renamed to sth else) is not too much code and the usage is self-explanatory. It's wrapping a few file operations. I will actually recommend we remove this interface design here. You don't need an interface here. It's fine to just have a concrete class `BlockWorkerMetaStore` which handles reading and writing cluster ID from a file.

Reply:
Storing the metadata(current is ClusterId) in a file makes it easier for workers to store more metadata, The current interface is relatively simple, I think there is an interface that may be more extensible.